cmd/soroban-cli/src/commands/contract/alias/add.rs

```diff
@@ -0,0 +1,84 @@
+use std::fmt::Debug;
+
+use clap::{command, Parser};
+
+use crate::commands::{config::network, global};
+use crate::config::locator;
+use crate::print::Print;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config_locator: locator::Args,
+
+    #[command(flatten)]
+    network: network::Args,
+
+    /// The contract alias that will be removed.
+    pub alias: String,
+
+    /// Overwrite the contract alias if it already exists.
+    #[arg(long)]
+    pub overwrite: bool,
+
+    /// The contract id that will be associated with the alias.
+    #[arg(long = "id")]
+    pub contract_id: String,
```

Comment:
[`stellar_strkey::Contract` implements `FromStr`](https://docs.rs/stellar-strkey/0.0.8/stellar_strkey/struct.Contract.html#impl-FromStr-for-Contract). Which means you can use it here and it will validate the contract id.
```suggestion
    pub contract_id: stellar_strkey::Contract,
```